docs/fundamentals/code-analysis/quality-rules/ca1854.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "CA1854: Prefer the 'IDictionary.TryGetValue(TKey, out TValue)' method"
+description: "Learn about code analyzer rule CA1854 - Prefer the 'IDictionary.TryGetValue(TKey, out TValue)' method"
+ms.date: 06/1/2022
+ms.topic: reference
+f1_keywords:
+    - CA1854
+helpviewer_keywords:
+    - CA1854
+author: CollinAlpert
+ms.author: CollinAlpert
+dev_langs:
+- CSharp
+- VB
+---
+
+# CA1854: Prefer the `IDictionary.TryGetValue(TKey, out TValue)` method
```

Comment:
@gewarren Are backticks okay in headings?

Reply:
Yes I think so - you can see them used here: https://docs.microsoft.com/en-us/dotnet/standard/base-types/changing-case#toupper-method